makefiles/boards/alias.inc.mk

```diff
@@ -0,0 +1,9 @@
+ALIAS = linux phynode
+
+# this basically implements a look-up table, is there a nicer way in make?
+ifneq (,$(filter phynode,$(BOARD)))
+  BOARD := pba-d-01-kw2x
+endif
+ifneq (,$(filter linux,$(BOARD)))
```

Comment:
- Ditch the filter call, BOARD is a single value, not a list.
 - Make this an else if

`else ifneq (linux, $(BOARD))`

Reply:
yes, makes sense -> gotten so used to the filter thing that I don't even question it anymore as it seems :-)